src/main/java/frc/robot/RobotContainer.java

```diff
@@ -202,6 +202,9 @@ private void configureBindings() {
     m_driverController
         .a()
         .whileTrue(alignToPose(() -> EagleUtil.getCachedReefPose(drivetrain.getState().Pose)));
+    m_driverController
```

Comment:
This will fail.  As the robot gets closer to the second-closest reef pose, it will become the closest one.
When we pick the target pose, we should not switch that until the drive lets go of the button

Reply:
I was wrong.   The closest pose is only picked once when the command starts, so this won't happen